db/revision_cache_test.go

```diff
@@ -1488,13 +1485,22 @@ func TestRevCacheOnDemandImport(t *testing.T) {
 	docID := "doc1"
 	revID, _, err := collection.Put(ctx, docID, Body{"ver": "1"})
 	require.NoError(t, err)
+
+	testCtx, testCtxCancel := context.WithCancel(base.TestCtx(t))
```

Comment:
```suggestion
	ctx, testCtxCancel := context.WithCancel(ctx)
```

I think it makes conceptual sense to me to not worry about which context to use then we can always just use the one known by `ctx` if we expand the test?